nmclicktests/environment.js

```diff
@@ -218,7 +218,7 @@ class Environment {
     return this.getPort()
       .then(() => this.startServer())
       .then(() => this.ensureStarted())
-      .catch((err) => { throw new Error("Cannot confirm ungit start!!"); })
+      .catch((err) => { console.log(err); throw new Error("Cannot confirm ungit start!!", err); })
```

Comment:
In JavaScript there is no `Error` constructor which takes the "inner" / "previous" error.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error
https://nodejs.org/api/errors.html#errors_new_error_message